bylaws/2-cf-membership.md

```diff
@@ -6,12 +6,20 @@ weight: 2
 
 The Commonhaus Foundation (CF) is a membership organization. The following section outlines the different types of membership and the process for becoming a member.
 
+- [Rights and requirements](#rights-and-requirements)
 - [Project Leaders and Code Owners](#project-leaders-and-code-owners)
 - [General Members](#general-members)
 - [Membership Dues and Renewal](#membership-dues-and-renewal)
 - [Inactive or Retired Members](#inactive-or-retired-members)
 - [Suspension and Revocation of Membership](#suspension-and-revocation-of-membership)
 
+## Rights and requirements
+
+Commonhaus Foundation members are natural persons, and join as individuals.
+This excludes in particular legal entities, which are represented exclusively through the means of the [advisory board][].
```

Comment:
I'd suggest a slight reword from "represented exclusively through the means of the advisory board" to "exclusively represented as part of the advisory board".

Reply:
Done, thanks